docs/monitoring/faq/webhooks.mdx

```diff
@@ -0,0 +1,79 @@
+# How to use webhooks
```

Comment:
nit: would title to "How to use webhooks with automations" and also add an entrypoint in `faq/index.html`